orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -67,6 +75,14 @@ def __init__(self, module, debug_options: DebugOptions):
 
         super(GraphExecutionManager, self).__init__(module._original_module)
 
+        # Debug flags
+        self._debug_options = debug_options
+
+        # Fallback configuration must be in the beginning to catch initialization errors
+        # Training and Evaluation mode can override this setting independently of each other
+        self._fallback_manager = _FallbackManager(policy=_FallbackPolicy.FALLBACK_DISABLE,
```

Comment:
>_FallbackPolicy.FALLBACK_DISABLE, [](http://example.com/codeflow?start=57&length=33)

let's target to turn this on by default in 1.9 release. 

Also think about how model-shipping team can try this out in their models. Baiju's interncal config would help.

Reply:
For the very first merge I disabled it but created a temporary `ORTModule(model, fallback_policy=None)` argument. But we can turn it on by default too

Reply:
OK.